bin/generate-coverage.sh

```diff
@@ -1,16 +1,21 @@
-#!/usr/bin/env nix-shell
-#!nix-shell -i bash --pure --packages cacert grcov rustup
+#!/usr/bin/env bash
 #shellcheck shell=bash
 
 set -eu
 
 # Create temporary working directory
-readonly WORKING_DIR="$(mktemp --directory)"
+WORKING_DIR="$(mktemp --directory)"
+readonly WORKING_DIR
```

Comment:
I'm not sure I approve of this stylistic change :smile:

Reply:
In the face of an unyielding linter, I, a seasoned programmer, was forced to my knees! "There is no other way!" the dreaded shellcheck boomed, its voice echoing through the halls of code. With a heavy heart, I surrendered, forever marked by this digital duel.

Reply:
Hehe :slightly_smiling_face: 

I'm curious which Shellcheck rule this violates, though. I've never seen it complain about that before...

Reply:
```
❯ nix log /nix/store/m66lfsaakg49psaq17991s13ayisbx21-generate-coverage.drv
In /nix/store/zx8hq1z1rp9nikksfdzkggmrcpqhinmk-generate-coverage/bin/generate-coverage line 14:
readonly WORKING_DIR="$(mktemp --directory)"
         ^---------^ SC2155 (warning): Declare and assign separately to avoid masking return values.

For more information:
  https://www.shellcheck.net/wiki/SC2155 -- Declare and assign separately to ...
```

Reply:
Thankfully, now, it is part of our development shell, so it will be checked constantly and in a reproducible way!